tool/missing-baseruby.bat

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
Do you intend to use this on Windows too?
But `#!` is not a valid command on Windows.
Usually shebang for `/bin/sh` is not necessary. 

Instead, `@echo off` will be better.
```suggestion
: "
@echo off
: "
```

The `:` is an invalid label to be ignored up to end of line on `cmd.exe`, and the command nothing to do on `sh` and
the 2nd and 3rd lines are its unused argument.